src/mono/cmake/configure.cmake

```diff
@@ -79,6 +80,21 @@ ac_check_funcs (
   gethrtime read_real_time gethostbyname gethostbyname2 getnameinfo getifaddrs
   access inet_ntop Qp2getifaddrs)
 
+check_c_source_compiles(
+  "
+  #include <zlib.h>
+  int main(void)
+  {
+    #if defined(ZLIB_VERNUM) && (ZLIB_VERNUM >= 0x1230)
+    #else
+    #error No good zlib found
+    #endif
+    return 0;
```

Comment:
This is copied from

https://github.com/mono/mono/blob/a791978a6e4312698137be451c648b650a8c8663/configure.ac#L728-L732

but I'm not sure if that's necessary.